README.org

```diff
@@ -242,33 +246,38 @@ filesystem, it is created. However, just the path is created, the files
 
 *** Loading the =custom.el= file
 
-    Rational Emacs loads the =custom.el= file during the initialization
-    process. As noted in the previous section, the =custom-set-variables= values
-    are in the =SET for current session only= state. Those values are saved to
-    the =custom.el= file just before the file is loaded after the initialization
-    process is finished. Any values set through the Customization UI or through
-    the other work flows, for example using the =org-agenda-to-front= or
-    =org-remove-file= which write to the =custom-set-variables= form, are
-    preserved as well.
+    When ~rational-load-custom-file~ is non-nil, the =custom.el= file is loaded
+    /after/ the initialization process, including after the user =config.el= is
+    loaded. By default, no custom file is loaded.
+
+    The =custom-set-variables= values (as set in =init.el=) are in the =SET for
```

Comment:
Did you mean ...

> the =customize-set-variable= values (as set in =init.el=) ...

because `custom-set-variables` is the form generated by Emacs when saving them to the `custom-file` (aka `custom.el` in our case).

Reply:
Thanks for catching that. I changed the wording.